zaza/openstack/charm_tests/test_utils.py

```diff
@@ -89,7 +89,53 @@ def audit_assertions(action,
             assert value == "PASS", "Unexpected failure: {}".format(key)
 
 
-class BaseCharmTest(unittest.TestCase):
+class CharmTestAssertions:
+    """Custom assertions to support zaza testings."""
+
+    def assertActionRanOK(self, action):
```

Comment:
This function doesn't use `self`.  Define using `@staticmethod`?

Reply:
I'll move it